src/neuroglancer/util/mouse_drag.ts

```diff
@@ -21,10 +21,9 @@ export function startRelativeMouseDrag(
   let {document} = initialEvent.view;
   let prevClientX = initialEvent.clientX, prevClientY = initialEvent.clientY;
   const mouseMoveHandler = (e: PointerEvent) => {
-    const deltaX = e.clientX - prevClientX;
-    const deltaY = e.clientY - prevClientY;
-    prevClientX = e.clientX;
-    prevClientY = e.clientY;
+    // clientX and screen are constant on pointerlock
```

Comment:
Behavior should depend on whether or not `document.pointerLockElement` is actually set to the expected element.